spec/components/candidate_interface/rejections_component_spec.rb

```diff
@@ -76,8 +90,7 @@
 
       result = render_inline(described_class.new(application_choice:, render_link_to_find_when_rejected_on_qualifications: true))
       expect(result.text).to include('View the course requirements on')
-      expect(result.css('.govuk-link')[0][:href]).to eq("#{course.find_url}#section-entry")
-      expect(result.css('.govuk-link')[0].text).to eq('Find postgraduate teacher training courses')
+      expect(page).to have_link('Find postgraduate teacher training courses', href: "#{course.find_url}#section-entry")
```

Comment:
Do these work if we changed this from `page` to `result`? I'm assuming they'll probably be testing the same thing 🤞